src/core/js/views/questionView.js

```diff
@@ -39,7 +39,15 @@ define([
       this._runModelCompatibleFunction('setupDefaultSettings');
       // Blank method for setting up questions before rendering
       this.setupQuestion();
+    }
+
+    getCanShowMarking() {
+      if (this.model.get('_onlyShowFinalMarking')) {
+        var attemptsLeft = (this.model.get('_attemptsLeft')) ? this.model.get('_attemptsLeft') : this.model.get('_attempts');
```

Comment:
Logic is sound, confused me for a bit though, perhaps something like:
```js
const attemptsLeft = this.model.get('_attemptsLeft');
const hasMultipleAttempts = Boolean(this.model.get('_attempts'));
const isFinalAttempt = !hasMultipleAttempts ||(attemptsLeft === 0);
```

What I've written is obviously all about face, but making the implied more explicit would be cool.